### ISSUE TITLE
Using `imports` results in tokens being transformed multiple times

### ISSUE THREAD

**Issue by shaunbent** [salesforce-ux/theo#132]
When using the `imports` feature to pull in tokens which exist in other files, the tokens are being processed twice. The second time the tokens are processed, they transform the result of the first time the tokens are processed and not the source original files.

Whilst this doesn't seem to cause any problems for most tokens/transforms this does break the Android 8 Digit HEX transform, as this relies on a regex and a `replace`. This results in an invalid output as the `replace` regex is called on an already transformed token.

**Example**

```yaml
# tokens.yml
props:
  COLOR_GREEN:
    category: colors
    type: color
    value: "#1DB954"

imports:
  - ./colors.yml
```

```yaml
# colors.yml
props:
  COLOR_GREEN_NESTED:
    category: colors
    type: color
    value: '#1DB954'

```

**Expected Output**
```xml
<?xml version="1.0" encoding="utf-8"?>
<resources>
  <color name="COLOR_GREEN_NESTED" category="colors">#ff1db954</color>
  <color name="COLOR_GREEN" category="colors">#ff1db954</color>
</resources>
```

**Actual Output**
```xml
<?xml version="1.0" encoding="UTF-8"?>
<resources>
  <property name="COLOR_GREEN_NESTED" category="colors" value="#54ff1db9"/>
  <property name="COLOR_GREEN" category="colors" value="#ff1db954"/>
</resources>
```

The the `color/hex8argb` transform is working as expected, the issue is somewhere else in the code, I would assume around how `imports` are handled.

I've tested this with a few versions and this isn't a problem for `5.0.0` but does effect `6.0.0-beta.7` and above.

**Comment by shaunbent** [comment]
@aputinski

**Comment by aputinski** [comment]
Hm.. I'll take a look today.

**Comment by shaunbent** [comment]
Hey @aputinski wondered if you had any thoughts on this issue?

If you could give me a few pointers of where to look I could take a look at fixing this myself and opening a PR.

Sorry to bother you.

**Comment by aputinski** [comment]
I just published `6.0.0-beta.10` which should fix the issue!

**Comment by shaunbent** [comment]
Amazing, thank you so much for the help with this. Massive appreciated. 🎉
